setup.py

```diff
@@ -46,12 +46,10 @@
     ],
     packages=find_packages(exclude=["tests", "tests.*", "tests/*"]),
     include_package_data=True,
-    package_data={
-        "version": [os.path.join("graspologic", "version", "version.txt")]
-    },
+    package_data={"version": [os.path.join("graspologic", "version", "version.txt")]},
     install_requires=[
         "networkx>=2.1",
-        "numpy>=1.8.1",
+        "numpy>=1.19.2",
```

Comment:
I changed this because `np.delete` behaved differently on earlier versions of numpy (you couldn't index by negative numbers previously). If we want, I can track down the specific version of numpy where that changed and have the requirement be that instead.

Reply:
this seems a bit drastic (relatively big change in what we are compatible with for a small function). wondering about A) using indexing instead, if performance is comparable B) converting indices to positive ourselves or C) just not allowing negative indexing for this func. thoughts?

Reply:
tracked down where behavior was changed, looks relatively recent: https://github.com/numpy/numpy/pull/15804
I'll play around with some other solutions, I agree that changing the numpy version requirement is a little drastic -- although on the other hand, is there a good reason to allow older versions of numpy?

Reply:
see f60a358 and 1052bdb